### ISSUE TITLE
Control Arduino slave from RPi?

### ISSUE THREAD

**Issue by epipheus** [hybridgroup/artoo#166]
Is it possible to control an Arduino slave via RPi both of which can leverage your DSL?

**Comment by deadprogram** [comment]
Yes, you can use the TTY interface on the Raspi to control an Arduino.

**Comment by epipheus** [comment]
Is this documented at all?

**Comment by deadprogram** [comment]
It is just artoo-arduino as normally used from Linux https://github.com/hybridgroup/artoo-arduino
